department-of-veterans-affairs/caseflow
# Remove unnecessary DB seeding, add Queue link to admin page, fix dev document count bug

Resolves #https://github.com/department-of-veterans-affairs/caseflow/issues/5077

### Testing Plan
1. Run `rake db:seed RAILS_ENV=local` `rake local:vacols:seed RAILS_ENV=local`
2. Observe that the fake users are gone from the Test Users page
3. Test that Reader and Queue still work as normal
4. Other apps will need additional work to be fully testable with FACOLS — Certification, Dispatch, and Hearing Prep all need FACOLS data improved
5. Previous to this PR, visiting `/queue` caused the stubbed document count call to fail every time with a 500. Now, it returns a document # (a fake one, though).
6. Test logging out and logging in. Still works.
